src/main/java/com/backendoori/ootw/service/MiniOImageServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.backendoori.ootw.service;
+
+import com.backendoori.ootw.config.MiniOConfig;
+import io.minio.GetPresignedObjectUrlArgs;
+import io.minio.MinioClient;
+import io.minio.PutObjectArgs;
+import io.minio.http.Method;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.multipart.MultipartFile;
+
+
+@Slf4j
+@Controller
+@RequiredArgsConstructor
+public class MiniOImageServiceImpl implements ImageService {
```

Comment:
Impl은 보통 단일 인터페이스, 단일 구현체에서 많이 사용해서 `MiniOImageService`로 클래스 명을 가져가면 좋을 것 같아요

```suggestion
public class MiniOImageService implements ImageService {
```